### DISCUSSION TITLE
Multilanguage CV.pdf

### DISCUSSION THREAD

**Discussion by saguileran** [george-gca/multi-language-al-folio#9] [Q&A] [answered]
I want to add another pdf file of my CV but in another language. I am creating a new folder at asst/pdf/example_pdf but it does not work. Do u know how can I solve this problem?

**Comment by george-gca** [comment]
I don't understand, you want to exhibit both PDFs in one language page? Like in the english version of the cv page have a PDF for more than one language?

**Reply by saguileran** [reply]
No, I want to have 2 different PDF files in the CV PDF (related image below), because I have two languages so I want to exhibit my CV.pdf en two different languages

![image](https://github.com/george-gca/multi-language-al-folio/assets/42812846/71a50401-23be-4840-82ef-6428c3227c3b)

**Reply by george-gca** [reply, marked as answer]
The current solution actually handles this. If you check the demo site in [english](https://george-gca.github.io/multi-language-al-folio/cv/), it points to a file `https://george-gca.github.io/multi-language-al-folio/assets/pdf/en/example_pdf.pdf`. Now, if you change the language to [pt-br](https://george-gca.github.io/multi-language-al-folio/pt-br/cv/), now it points to `https://george-gca.github.io/multi-language-al-folio/pt-br/assets/pdf/pt-br/example_pdf.pdf`. The current requirement is that the files have the same name, but they are stored each in a directory under `assets/pdf/LANG/`.

**Reply by luissequeira** [reply]
Hi @george-gca, I tested this on the demo site and the English version works fine but the `pt-br` link to download the file gives me the "page not found" error [https://george-gca.github.io/multi-language-al-folio/pt-br/assets/pdf/pt-br/example_pdf.pdf](https://george-gca.github.io/multi-language-al-folio/pt-br/assets/pdf/pt-br/example_pdf.pdf). I'm getting the same error in my local.

**Reply by george-gca** [reply]
You are right, there was a bug with it. When I implemented to avoid localization of assets, I forgot to update the localized pdf path. Fixed it now.

**Reply by saguileran** [reply]
Thank you George, now the links are working perfectly!!
